vic/drivers/shared_image/src/nc_utils.c

```diff
@@ -0,0 +1,58 @@
+/******************************************************************************
+ * @section DESCRIPTION
+ *
+ * Functions related to opening and closing netCDF files.
+ *
+ * @section LICENSE
+ *
+ * The Variable Infiltration Capacity (VIC) macroscale hydrological model
+ * Copyright (C) 2016 The Computational Hydrology Group, Department of Civil
+ * and Environmental Engineering, University of Washington.
+ *
+ * The VIC model is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *****************************************************************************/
+
+#include <vic_driver_shared_image.h>
+
+/******************************************************************************
+ * @brief    Open a netCDF file
+ *****************************************************************************/
+int
+open_nc(char *nc_name)
```

Comment:
Change function names - this is too close to `nc_open` and `nc_close`. Also I don't quite understand why we need the wrapper functions, since not much happens here other than a call to `nc_open`. So

a) motivate why we need these functions
b) if we need them, change their names.